Configuration/Icons.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use TYPO3\CMS\Core\Imaging\IconProvider\SvgIconProvider;
+
+// Register SVG Icon Identifier
+$svgIcons = [
+    'ext-daycarecenters-careform' => 'careform.svg',
+    'ext-daycarecenters-district' => 'district.svg',
+    'ext-daycarecenters-holder' => 'holder.svg',
+    'ext-daycarecenters-kita' => 'kita.svg',
+    'ext-daycarecenters-telephone' => 'telephone.svg',
+];
+
+$configuredIcons = [];
```

Comment:
Don't use local variables in these files. Local variables may interfere with local variables of other Icon.php files. If you want to use local variables please wrap them into a call_user_func() like we do in ext_localconf.php or TCA/Override files.